src/v2/options.ts

```diff
@@ -246,8 +267,8 @@ export function optionsToTriggerAnnotations(
     opts,
     'availableMemoryMb',
     'memory',
-    (mem: MemoryOption) => {
-      return MemoryOptionToMB[mem];
+    (mem: MemoryOption | Expression<number>): number | string => {
+      return isMemoryOption(mem) ? MemoryOptionToMB[mem] : ExprString(mem);
```

Comment:
Make `Expression` implement `toString` and use that. We need `toString` in order to make string interpolation work anyway.

Reply:
Outdated.